src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -328,6 +361,35 @@ export class AccessibleView extends Disposable {
 		this._instantiationService.createInstance(AccessibleViewSymbolQuickPick, this).show(this._currentProvider);
 	}
 
+	calculateCodeBlocks(markdown: string): void {
+		if (!this._currentProvider) {
+			return;
+		}
+		if (this._currentProvider.options.language && this._currentProvider.options.language !== 'markdown') {
+			// Symbols haven't been provided and we cannot parse this language
+			return;
+		}
+		const lines = markdown.split('\n');
+		this._codeBlocks = [];
+		let inBlock = false;
+		let startLine = 0;
+
+		lines.forEach((line, i) => {
+			let languageId: string | undefined;
+			if (!inBlock && line.startsWith('```')) {
```

Comment:
You might want to try to use the real markdown parser for this since codeblocks can be triggered by other characters and contain '```' sequences. I guess that would be some effort to count lines for the source though.

Reply:
As you said, I think that would be tricky to count the lines